deployment/quick-start/services-configuration.yaml.template

```diff
@@ -41,6 +41,10 @@ cluster:
     # Must be lower case, e.g., regsecret.
     secret-name: pai-secret
 
+  api-server:
+    # use tls for prometheus access
+    tls_access: false
```

Comment:
maybe should set a default value in pai object module to false, without default value, user with previous config may get exception in expanding template of prometheus?

Reply:
added at pai object model